cvat-canvas/src/typescript/canvasView.ts

```diff
@@ -2877,6 +2877,9 @@ export class CanvasViewImpl implements CanvasView, Listener {
             const shapeView = window.document.getElementById(`cvat_canvas_shape_${clientID}`);
             if (shapeView) shapeView.classList.remove(this.getHighlightClassname());
         });
+        const redrawMasks = (highlightedElements.elementsIDs.length !== 0 ||
+            this.highlightedElements.elementsIDs.length !== 0);
```

Comment:
As I see below. We redraw all drawn masks. Do we really need to do it?
Why not redraw only masks which are in highligted elements array?

Reply:
There are some cases when we need to redraw masks outside of highligted elements array. 
1. Mask was in previous highligted elements array and needs to be redrawn with 'SHADED_COLOR'
2. Highligted elements array is empty and we need to redraw all `highlighted` and `shaded` masks with normal color.

Reply:
That does not answer the main question. 
Do we need to redraw all drawn masks (let's say we have hundred of them on frame) and only one will be / was highlighted?